Framework/DataHandling/src/SetSample.cpp

```diff
@@ -31,6 +31,7 @@
 #include <Poco/Path.h>
 #include <boost/algorithm/string/case_conv.hpp>
 #include <boost/algorithm/string/predicate.hpp>
+#include <boost/range/algorithm/transform.hpp>
```

Comment:
I don't think you need this include. Might be a feature from the original PR for this change but compiles for me without it